frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -1048,6 +1050,12 @@ export const eventUsageLogic = kea<eventUsageLogicType>([
         reportFailedToCreateFeatureFlagWithCohort: ({ detail, code }) => {
             posthog.capture('failed to create feature flag with cohort', { detail, code })
         },
+        reportFeatureFlagCopySuccess: () => {
```

Comment:
```suggestion
        reportFeatureFlagProjectCopySuccess: () => {
```

and then similarly for “feature flag project copied”

Maybe the name can be a bit more indicative of what it’s doing? You can also copy feature flags like the key and such so that would get confusing.